app/services/edge_cache/bust/fastly.rb

```diff
@@ -1,24 +1,37 @@
 module EdgeCache
   class Bust
     class Fastly
+      # [@forem/systems] Fastly-enabled Forems don't need "flexible" domains.
       def self.call(path)
-        # TODO: (Alex Smith) - It would be "nice to have" the ability to use the
-        # Fastly gem here instead of custom API calls.
+        api_key = ApplicationConfig["FASTLY_API_KEY"]
+        return fastly_purge(api_key, path) if FeatureFlag.enabled?(:fastly_http_purge)
 
-        # @forem/systems Fastly-enabled forems don't need "flexible" domains.
-        HTTParty.post(
-          "https://api.fastly.com/purge/https://#{URL.domain}#{path}",
-          headers: {
-            "Fastly-Key" => ApplicationConfig["FASTLY_API_KEY"]
-          },
-        )
-        HTTParty.post(
-          "https://api.fastly.com/purge/https://#{URL.domain}#{path}?i=i",
-          headers: {
-            "Fastly-Key" => ApplicationConfig["FASTLY_API_KEY"]
-          },
-        )
+        fastly_post(api_key, path)
       end
+
+      def self.fastly_post(api_key, path)
+        urls(path).map do |url|
+          HTTParty.post("https://api.fastly.com/purge/#{url}", headers: { "Fastly-Key" => api_key })
+        end
+      end
+      private_class_method :fastly_post
+
+      def self.fastly_purge(api_key, path)
+        fastly = ::Fastly.new(api_key: api_key)
+
+        urls(path).map do |url|
+          fastly.purge(url)
+        end
+      end
+      private_class_method :fastly_purge
+
+      def self.urls(path)
+        [
+          "https://#{URL.domain}#{path}",
```

Comment:
Could we use our helper here?

```suggestion
          URL.url(path),
```